lib/instrumentation/shimmer.js

```diff
@@ -32,7 +32,7 @@ function logger () {
 }
 
 function isFunction (funktion) {
-  return funktion && {}.toString.call(funktion) === '[object Function]'
+  return typeof funktion === 'function'
```

Comment:
@Qard Wow, nice find!

This was also updated in the upstream `shimmer` module last year: https://github.com/othiym23/shimmer/commit/ec15ba2bf2cffc25622fca0d277ea68f0ae2d8ee

I wonder why `typeof` wasn't used originally.

Why was this making knex fail btw? I would love to know what knex did to fail this check 🤔 Did they use async or generator functions?

Reply:
They started using async functions.